test/e2e/apimachinery/chunking.go

```diff
@@ -122,6 +122,29 @@ var _ = SIGDescribe("Servers with support for API chunking", func() {
 		gomega.Expect(list.Items).To(gomega.HaveLen(numberOfTotalResources))
 	})
 
+	ginkgo.It("should chunk lists of Pods", func() {
+		ns := f.Namespace.Name
+		c := f.ClientSet
+		client := c.CoreV1().PodTemplates(ns)
+		loopCount := int64(0)
+		for {
+			opts := metav1.ListOptions{}
```

Comment:
Don't you need to set opts to use the continue token from the last list iteration?

Reply:
I have updated the test to use what you suggested in https://github.com/ii/kubernetes/commit/cee471ad3a972b37fd928aa6af1891d8323b75df#diff-800e2a08a32f7d269d6af9e0c51c7d6dR137; It will only use it, if it's had it's first iteration.